app/src/main/res/xml/preferences.xml

```diff
@@ -55,7 +55,7 @@
         android:title="@string/preferences_category_privacy"
         app:iconSpaceReserved="false">
         <androidx.preference.Preference
-            android:icon="@drawable/ic_tracking_protection_enabled"
+            app:icon="@drawable/ic_tracking_protection_enabled"
```

Comment:
I think preferences.xml should automatically use compat with no changes. Does the settings screen still show the icon?

Reply:
Yeah u right I can't reproduce idk why I thought this was broken too 🤷‍♀